envs/monkey_maker/README.md

```diff
@@ -0,0 +1,57 @@
+# Monkey maker
+
+## About
+
+Monkey maker is an environment on AWS that 
+is designed for monkey binary building. 
+This environment is deployed using terraform scripts
+located in this directory.
+
+## Setup
+
+To setup you need to put `accessKeys` file into `./aws_keys` directory.
+
+Contents of `accessKeys` file should be as follows:
+
+```
+[default]
+aws_access_key_id = <...>
+aws_secret_access_key = <...>
+```
+Also review `./terraform/config.tf` file.
+
+Launch the environment by going into `terraform` folder and running
+```angular2html
+terraform init
+terraform apply
+```
+
+### Usage
+
+To login to windows use Administrator: %HwuzI!Uzsyfa=cB*XaQ6xxHqopfj)h) credentials
+
+You'll find docker files in `/home/ubuntu/docker_envs/linux/...`
```

Comment:
Does this make sense? Why are we not pulling latest docker files from internet?

Reply:
We can skip the build step

Reply:
You want me to add that we also need to pull the latest changes before building? Isn't that kind of self explanatory, that if you want to build monkey of a certain branch you need to pull that branch? If we skip the build step, user will have to check image name manually?